packages/ckeditor5-link/docs/features/link.md

```diff
@@ -75,7 +75,7 @@ There are two types of link decorators you can use:
 
 ### Demo
 
-In the editor below, all **external** links get the `target="_blank"` and `rel="noopener noreferrer"` attributes ([automatic decorator](#adding-attributes-to-links-based-on-predefined-rules-automatic-decorators)). Click a link and edit it {@icon @ckeditor/ckeditor5-core/theme/icons/pencil.svg Edit link} to see that you can control the `download` attribute of specific links using the switch button in the editing balloon ([manual decorator](#adding-attributes-to-links-using-the-ui-manual-decorators)). Take a look at the editor data below (updated live) to see the extra link attributes.
+In the editor below, all **external** links get the `target="_blank"` and `rel="noopener noreferrer"` attributes ([automatic decorator](#adding-attributes-to-links-based-on-predefined-rules-automatic-decorators)). Click a link and check its properties {@icon @ckeditor/ckeditor5-core/theme/icons/settings.svg Link properties} to see that you can control the `download` attribute of specific links using the switch button in the editing balloon ([manual decorator](#adding-attributes-to-links-using-the-ui-manual-decorators)). Take a look at the editor data below (updated live) to see the extra link attributes.
```

Comment:
Same, the "downloadable" option is in the "edit link" button. Will this change? If so, I rest my case.

Reply:
Hmm... it seems like your docs build on an older version(?) I see this new UI.

<img width="749" alt="image" src="https://github.com/user-attachments/assets/ba34549d-e6aa-4ea0-9117-68a017d4d983">

Reply:
That's droll, I pulled the latest version of this PR. Is there some corresponding branch on `commercial` for this?

Reply:
No, nothing on the commercial. Tbh, I don't know why. I see version 44 in the URL: `http://127.0.0.1:8080/ckeditor5/44.0.0/features/link.html`.

Reply:
`44.0.0`, too. Just my environment again, probably, sigh. Let's wait till it's merged and see.

Reply:
That's strange... Maybe you should `checkout` the epic first? Or reinstall? Hard to tell.